WooCommerce/Classes/ViewRelated/Coupons/Add and Edit Coupons/AddEditCoupon.swift

```diff
@@ -17,48 +17,55 @@ struct AddEditCoupon: View {
         NavigationView {
             GeometryReader { geometry in
                 ScrollView {
-                    VStack (alignment: .leading) {
+                    VStack (alignment: .leading, spacing: 0) {
                         Group {
                             ListHeaderView(text: Localization.headerCouponDetails.uppercased(), alignment: .left)
 
                             Group {
-                                TitleAndTextFieldRow(title: Localization.couponAmountPercentage,
-                                                     placeholder: Localization.couponAmountPercentage,
+                                TitleAndTextFieldRow(title: viewModel.amountText,
+                                                     placeholder: "0",
                                                      text: $viewModel.amountField,
-                                                     editable: false,
+                                                     editable: true,
                                                      fieldAlignment: .leading,
-                                                     keyboardType: .asciiCapableNumberPad)
+                                                     keyboardType: .decimalPad)
                                 Divider()
                                     .padding(.leading, Constants.margin)
+
+                                Text(viewModel.amountSubtitleText)
+                                    .subheadlineStyle()
+                                    .padding(.horizontal, Constants.margin)
                             }
+                            .padding([.bottom], Constants.verticalSpacing)
```

Comment:
Nit: we might not need to put the single edge in an array.
```suggestion
                            .padding(.bottom, Constants.verticalSpacing)
```

Reply:
I see a lot of this in the file 😅 Please feel free to update them all to make them look cleaner.